cdserver/app/controllers/diagnostics_controller.rb

```diff
@@ -1,4 +1,5 @@
 class DiagnosticsController < ApplicationController
+  before_action :authenticate_user
 
   def info
```

Comment:
neither /info nor /healthcheck should require authentication
however, all the other endpoints will